hack/make/binary

```diff
@@ -62,3 +62,18 @@ echo "Created binary: $DEST/$BINARY_FULLNAME"
 ln -sf "$BINARY_FULLNAME" "$DEST/docker$BINARY_EXTENSION"
 
 hash_files "$DEST/$BINARY_FULLNAME"
+
+# Add nested executables to bundle dir so we have complete set of
+# them available, but only if the native OS/ARCH is the same as the
+# OS/ARCH of the build target
+if [ "$(go env GOOS)/$(go env GOARCH)" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then
+	echo "Copying nested executables into $DEST"
+	(set -x
+	if [ -x /usr/local/bin/runc ]; then
+		cp /usr/local/bin/runc $DEST/
```

Comment:
@jfrazelle what's the issue you see with having them in the other dirs?  There's some consistency to having runc/containerd/... sitting next to the docker exe, no?

Are you asking why we need to copy runc/containerd/... ?  The output of "make binary" needs to have the complete set of exes needed to run docker. It might be nice to have runc place its exe into the bundles dir, but I'm not sure how clean it would be to pass the bundles dir down into runc's build scripts.